src/Microsoft.Health.Fhir.Core/Configs/CoreFeatureConfiguration.cs

```diff
@@ -19,5 +19,10 @@ public class CoreFeatureConfiguration
         /// Gets or sets a value indicating whether Transaction is enabled or not.
         /// </summary>
         public bool SupportsTransaction { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether Validate is enabled or not.
+        /// </summary>
+        public bool SupportsValidate { get; set; }
```

Comment:
I'm not sure if there's plan to expose this via the `/metadata` endpoint, but if not, then this setting could probably just move to the `features` settings property versus the `coreFeatures`

Reply:
I was unsure about what the difference between the two was. I put it in here since it was in the Core project's feature folder. I'll move it if it makes more sense elsewhere.

Reply:
Flag moved